api_types.ts

```diff
@@ -339,13 +339,18 @@ export interface ParamDef<T extends UnionType> {
   suggestedValue?: SuggestedValueType<T>;
 
   /**
-   * In certain contexts (like Coda Brain), Coda's default behavior is to sync *all* data available, in which case
-   * the {@link ParamDef.suggestedValue} will not be ideal, as most use cases will prefer efficiency
-   * over completeness. Use this field to specify a default value when Coda is syncing all data.
+   * In crawl syncs, where we want to load *all* data available, the {@link ParamDef.suggestedValue}
```

Comment:
Sorry if I'm misremembering the outcome, how come the property is "ingestionSuggestedValue" but the documentation frames it as "crawl syncs"? Would have thought the documentation would match the property name.

Reply:
Sorry, yes, updated comments to remove all references to "crawl sync"